### PR TITLE
Remove irrelevant options from checklist in PULL_REQUEST_TEMPLATE.md …

### LINKED ISSUE #100: Bug: Remove irrelevant options from checklist in PULL_REQUEST_TEMPLATE.md
### Describe the bug
There is a option ` I have written Kotlin Docs whenever is applicable` under the checklist in PULL_REQUEST_TEMPLATE.md. (I think due to copy-pasting the file from some other project).

### To Reproduce
1. Go to https://github.com/anitab-org/open-source-programs-web/blob/develop/.github/PULL_REQUEST_TEMPLATE.md
2. See error

### Expected behavior
The option should be removed from the checklist.

### Desktop (please complete the following information):
 - OS: Any
 - Browser: Any
 - Version: Any

### PR DESCRIPTION
**Pull request by mehakkkk**
…#100

### Description
removed irrelevant option from the PULL_REQUEST_TEMPLATE.md file.

Fixes #100 

### Type of Change:

- Documentation

**Code/Quality Assurance Only**
- This change requires a documentation update (checklist update on PULL_REQUEST_TEMPLATE.md file)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

**Desktop (please complete the following information):
OS: Linux
Browser: Chrome
Version: Version 86.0.4240.198

### CODE REVIEW

**Comment by Rahulm2310** [comment]
@mehakkkk Just a request, if you want to work on a issue, please comment on the issue requesting a maintainer to assign you and wait until it is assigned to you. Please be patient. You will get ample chances to contribute. As a first timer, you should go through [Contribution Guidelines](https://github.com/anitab-org/open-source-programs-web/blob/develop/.github/CONTRIBUTING.md).

Cc : @isabelcosta

**Comment by mehakkkk** [comment]
@Rahulm2310 I apologize for this fretful behaviour. I am new to open source and getting acquainted with it. Will make sure to follow all the guidelines provided.

**Comment by codesankalp** [comment]
@mehakkkk Please comment on the issue and I will assign it to you.
And next time keep @Rahulm2310 point in mind while contributing to this organization.

**Review by codesankalp — APPROVED** [review]
LGTM @mehakkkk :tada:

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -32,12 +32,16 @@ Fixes #ISSUE
...
 **Code/Quality Assurance Only**
 - [ ] My changes generate no new warnings
 - [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
-- [ ] Any dependent changes have been published in downstream modules
\ No newline at end of file
+- [ ] Any dependent changes have been published in downstream modules
+
+**Desktop (please complete the following information):
```
Comment (Aaishpra) [inline .github/PULL_REQUEST_TEMPLATE.md:44]: No need for desktop and browser details to be included in PR templates, please remove this

**Review by Aaishpra — CHANGES_REQUESTED** [review]
@mehakkkk can you edit the PR description and remove the "..#100" from the top.

**Comment by Rahulm2310** [comment]
@mehakkkk Also, please remove irrelevant options from your PR description (under the checklist and code/quality assurance only).
Also, why `Code/Quality Assurance Only` list is present 2 times. Remove one.

**Review by ashwinijha6 — CHANGES_REQUESTED** [review]
@mehakkkk could you please mention the type `fix` in the PR title & remove the line `...#100` before the description

**Review by ashwinijha6 — CHANGES_REQUESTED** [review]
Also you could add this line in the `Checklist:`
* Update requirements.txt
What do you think @codesankalp ?

**Comment by codesankalp** [comment]
@ashwinijha6 There is no `requirements.txt` in this repo but yeah @mehakkkk can add the line for updating `package.json`.

**Comment by ashwinijha6** [comment]
> @ashwinijha6 There is no `requirements.txt` in this repo but yeah @mehakkkk can add the line for updating `package.json`.

Oh ya @codesankalp

**Review by ashwinijha6 — CHANGES_REQUESTED** [review]
Update Docusaurus documentation and the exported file at /docs folder
This could also be added @mehakkkk 
cc : - @codesankalp

**Comment by codesankalp** [comment]
@mehakkkk Are you going to work on it?
@Aaishpra @keshakaneria Should we close this pull request as there is no response from @mehakkkk?

**Comment by Aaishpra** [comment]
Yaa i think we should close this Pr and make the issue available for others.

**Comment by Aaishpra** [comment]
Closing this due to inactivity

**Pull request closed without merging**